Formula/weechat.rb

```diff
@@ -25,7 +25,7 @@ class Weechat < Formula
   depends_on "aspell" => :optional
   depends_on "lua" => :optional
   depends_on "python" => :optional
-  depends_on :ruby => ["2.1", :optional]
+  depends_on "ruby" => :optional
   depends_on :perl => ["5.3", :optional]
```

Comment:
`:perl is deprecated. Usage should be "perl"`

https://github.com/Homebrew/homebrew-core/pull/22703

cc @ilovezfs

Reply:
yup no need to fix it here